paket.dependencies

```diff
@@ -15,6 +15,8 @@ source https://api.nuget.org/v3/index.json
 
 nuget NuGet.CommandLine
 nuget StyleCop.Analyzers
```

Comment:
I know we didn't do this previously but I'd rather put analyzers in their own analysis group.

Reply:
Done.